src/main/java/org/commcare/formplayer/services/MenuSessionRunnerService.java

```diff
@@ -668,6 +668,10 @@ private NewFormResponse generateFormEntrySession(MenuSession menuSession) throws
                 formDefinitionService);
 
         NewFormResponse response = newFormResponseFactory.getResponse(formEntrySession);
+
+        String commandId = menuSession.getSessionWrapper().getCommand();
+        newFormResponseFactory.setTranslations(response, commandId);
+
```

Comment:
Why not init translations in ` newFormResponseFactory.getResponse` as before ?

Reply:
I thought this was cleaner to pass in `commandId` just where it's needed rather than pass into `NewFormResponse`

Reply:
I would think otherwise here as the response **_factory_** should be responsible to form complete response without cluttering the calling service with additional calls.